src/components/SpaceListItem/SpaceListItem.tsx

```diff
@@ -54,9 +63,16 @@ const SpaceListItem: FC<Props> = forwardRef(
               style={{ color: `var(--theme-text-team-${teamColor}-normal)` }}
               type="body-secondary"
               data-test="list-item-second-line"
+              aria-label={_secondLineArrayClean.join(', ')}
             >
-              {/* //TODO: change with dot divider when available */}
-              {_secondLineArray.join(' - ')}
+              {_secondLineArrayClean.map((secondLineContent, i) => {
+                const showDividerDot = i > 0;
+                return (
+                  <SecondLineElement key={`second-line-item-${i}`} showDividerDot={showDividerDot}>
+                    {secondLineContent}
+                  </SecondLineElement>
+                );
+              })}
```

Comment:
It is a minor thing but you can simplify this code a bit
```suggestion
              {_secondLineArrayClean.map((secondLineContent, i) => (
                  <SecondLineElement key={`second-line-item-${i}`} showDividerDot={i > 0}>
                    {secondLineContent}
                  </SecondLineElement>
                )
             )}
```